widgets/comctrls.pas

```diff
@@ -95,40 +95,46 @@   TCustomPageControl = class(TWinControl)
+    FPageIndex: nativeint;
     FPages: TJSArray;
     FShowTabs: boolean;
     FTabContainerElement: TJSHTMLElement;
     FTabHeight: smallint;
     FTabPosition: TTabPosition;
     FTabWidth: smallint;
     function GetActivePage: TCustomTabSheet;
-    function GetPage(const AIndex: NativeInt): TCustomTabSheet;
-    function GetPageCount: NativeInt;
+    function GetPage(const AIndex: nativeint): TCustomTabSheet;
+    function GetPageCount: nativeint;
     procedure SetActivePage(AValue: TCustomTabSheet);
     procedure SetMultiLine(AValue: boolean);
-    procedure SetPageIndex(AValue: NativeInt);
+    procedure SetPageIndex(AValue: nativeint);
     procedure SetShowTabs(AValue: boolean);
     procedure SetTabHeight(AValue: smallint);
     procedure SetTabPosition(AValue: TTabPosition);
     procedure SetTabWidth(AValue: smallint);
   protected
-    property TabIndex: NativeInt read FPageIndex write SetPageIndex;
+    property TabIndex: nativeint read FPageIndex write SetPageIndex;
   protected
     procedure Changed; override;
     function CreateHandleElement: TJSHTMLElement; override;
     function CreateTabContainerElement: TJSHTMLElement; virtual;
     procedure RegisterChild(AControl: TControl); override;
     procedure UnRegisterChild(AControl: TControl); override;
-    function CalcTabHeight: NativeInt; virtual;
-    function CalcTabWidth(const AText: string): NativeInt; virtual;
-    function CalcMaxTabWidth: NativeInt; virtual;
-    function CalcSumTabsWidth: NativeInt; virtual;
-    function IndexOfTab(const ACaption: string): NativeInt;
-    function RenderTab(const ACaption: string; const ALeft, ATop, AWidth, AHeight: NativeInt; const AEvent: JSValue): TJSHTMLElement; virtual;
-    function RenderTabActive(const ACaption: string; const ALeft, ATop, AWidth, AHeight: NativeInt; const AEvent: JSValue): TJSHTMLElement; virtual;
-    function RenderTabLeft(const ALeft, ATop, AWidth, AHeight: NativeInt; const AEvent: JSValue): TJSHTMLElement; virtual;
-    function RenderTabRight(const ALeft, ATop, AWidth, AHeight: NativeInt; const AEvent: JSValue): TJSHTMLElement; virtual;
+    function CalcTabHeight: nativeint; virtual;
+    function CalcTabWidth(const AText: string): nativeint; virtual;
+    function CalcMaxTabWidth: nativeint; virtual;
+    function CalcSumTabsWidth: nativeint; virtual;
+    function IndexOfTab(const ACaption: string): nativeint;
+    function RenderTab(const ACaption: string;
+      const ALeft, ATop, AWidth, AHeight: nativeint;
+      const AEvent: JSValue): TJSHTMLElement; virtual;
+    function RenderTabActive(const ACaption: string;
+      const ALeft, ATop, AWidth, AHeight: nativeint;
+      const AEvent: JSValue): TJSHTMLElement; virtual;
+    function RenderTabLeft(const ALeft, ATop, AWidth, AHeight: nativeint;
+      const AEvent: JSValue): TJSHTMLElement; virtual;
+    function RenderTabRight(const ALeft, ATop, AWidth, AHeight: nativeint;
+      const AEvent: JSValue): TJSHTMLElement; virtual;
```

Comment:
Would you please remove all these unrelated casing and line break changes? We can talk about the later in the context of a separate commit if the IDE is really pesky about it, but I don't want the former (for example the ```Menus``` unit at the top should be used as ```Menus``` as well instead of ```menus```)

Reply:
Ok. I will do it. 
JEDI code formatter :) (Ctrl+D). 
Hm... I have not made a PullRequest for PopupMenu, becouse it's not ready for merge with main repo.

Reply:
I'll probably have to publish settings for JCF... I personally don't use it and it's the first thing that gets uninstalled, cause I often accidentally use Ctrl + D and then the compiler's files (those especially!) get trashed up...

Reply:
Also you should put your changes in a separate branch and publish a Pull Request for that, cause otherwise all further changes you do in the branch will be part of the Pull Request as well (I take it you continued to use Master?)

Reply:
Yes, i using master branch. I have small expirence for using GIT. :( 
But i think github is a good place to get to know this better.

PS: And sorry for my English. :)